rest-api/flow/docs/grpc-api.md

```
@@ -735,644 +1042,1584 @@ scope entries. In-flight tasks are not cancelled.
 
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
 | id | [UUID](#v1-UUID) |  |  |
-| with_rack | [bool](#bool) |  |  |
+| read_only | [bool](#bool) |  |  |
+| name | [string](#string) |  |  |
+| description | [string](#string) |  |  |
+| enabled | [bool](#bool) |  |  |
+| event_type | [string](#string) |  |  |
+| actions | [EventRuleAction](#v1-EventRuleAction) | repeated |  |
+| created_at | [google.protobuf.Timestamp](#google-protobuf-Timestamp) |  |  |
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)